package.json

```diff
@@ -77,8 +77,13 @@
   },
   "jest": {
     "collectCoverage": true,
-    "collectCoverageFrom": ["webpack/**/*.js", "!webpack/**/bundle*"],
-    "coverageReporters": ["lcov"],
+    "collectCoverageFrom": [
+      "webpack/**/*.js",
+      "!webpack/**/bundle*"
+    ],
+    "coverageReporters": [
+      "lcov"
+    ],
```

Comment:
I'm not sure why but when I run `npm test` or `npm i` it changes the style here :/